packages/odyssey-react/src/components/Box/_box-text.scss

```diff
@@ -14,6 +14,10 @@
   color: var(--ColorTextBody);
 }
 
+.colorDanger {
+  color: var(--ColorTextDanger);
+}
+
```

Comment:
we can remove this CSS and the associated `color="danger"` prop now yes? Why is this still around?

Reply:
Im confused here. Why remove color danger? If you want anything other than color body, should you be using Text? Is that the only point of text?

Reply:
@arnoldsandoval-okta the abstraction right now is aiming to focus on two different use cases in the DOM, Text stuff and generalized Box like stuff (div, span with some margin maybe and a border).

The more font color styles you add to Box, the more ambiguous the API becomes. We want to design an API that is as unambiguous as possible because its easier to use for developers and to maintain for us. Does that help?

If you want to talk more about the API design we should catch up at Office Hours today, we can bring up the notes from our previous talk to help refresh us all.